packages/js/src/components/contentAnalysis/SeoAnalysis.js

```diff
@@ -191,15 +191,6 @@ class SeoAnalysis extends Component {
 		}
 		link = addQueryArgs( link, { context: locationContext } );
 
-		/*
-		 * We don't show the upsell in WooCommerce product pages when Yoast SEO WooCommerce plugin is activated.
-		 * This is because the premium assessments of the upsell are already loaded even when the Premium plugin is not activated.
-		 */
-		const contentType = wpseoAdminL10n.postType;
-		if ( this.props.isYoastSEOWooActive && contentType === "product" ) {
```

Comment:
If we're not using the prop `isYoastSEOWooActive` at all (which is great), we should consider removing it altogether, in these components, but also in the fills in `packages/js/src/components/fills/MetaboxFill.js`, `packages/js/src/components/fills/SidebarFill.js`, and `packages/js/src/elementor/components/fills/ElementorFill.js` 😄

Reply:
For context, this was [the PR ](https://github.com/Yoast/wordpress-seo/pull/18658)that introduced the prop